### PR TITLE
chore(geo): v6 migration for MapView and LocationSearch, updated to point to console-preview amplify package

### PR DESCRIPTION
**Pull request by thaddmt**
#### Description of changes
- Updates packages to point at console-preview tag since geo changes are not all on next yet
- Just putting it out for review right now, will not work until maplibre amplify package is released

#### Checklist

- [ ] Have read the [Pull Request Guidelines](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md)
- [ ] PR description included
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] `yarn test` passes and tests are updated/added
- [ ] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: examples/next/package.json [hunk 1]
```diff
@@ -10,13 +10,14 @@
...
   },
   "dependencies": {
+    "@aws-amplify/geo": "console-preview",
     "@aws-amplify/ui-react": "^5.3.1",
+    "@aws-amplify/ui-react-geo": "^1.0.2",
     "@aws-amplify/ui-react-liveness": "^2.0.8",
-    "@aws-amplify/ui-react-storage": "^2.3.1",
     "@aws-amplify/ui-react-notifications": "^1.0.13",
-    "@aws-amplify/ui-react-geo": "^1.0.2",
+    "@aws-amplify/ui-react-storage": "^2.3.1",
```
Comment (reesscot) [inline examples/next/package.json:18]: weird, why was this so out of date

File: packages/react-geo/package.json [hunk 2]
```diff
@@ -42,12 +42,13 @@
   "dependencies": {
     "mapbox-gl": "1.13.1",
     "maplibre-gl": "2.1.9",
-    "maplibre-gl-js-amplify": "^3.1.0",
+    "maplibre-gl-js-amplify": "3.0.0-thadd",
```
Comment (reesscot) [inline packages/react-geo/package.json:45]: best tag release name ever

File: packages/react-geo/src/components/MapView/MapView.tsx [hunk 3]
```diff
@@ -1,23 +1,11 @@
...
 import maplibregl from 'maplibre-gl';
 import { AmplifyMapLibreRequest } from 'maplibre-gl-js-amplify';
 import ReactMapGL from 'react-map-gl';
 import type { MapProps, MapRef, TransformRequestFunction } from 'react-map-gl';
-import { Amplify } from 'aws-amplify';
-import * as Auth from '@aws-amplify/auth';
-
-// Utility types for missing AmplifyConfig type, only includes Geo related key/values.
-// Note: these types should not be be used outside this file
-type AmplifyGeoOptions = {
```
Comment (reesscot) [inline packages/react-geo/src/components/MapView/MapView.tsx:11]: Where these types moved to the Geo JS category package?

**Review by reesscot — APPROVED** [review]

File: examples/next/package.json (hunk 1 above)
Comment (thaddmt) [reply examples/next/package.json:18]: Oh I think things just got reoredered so the versions look weird

File: packages/react-geo/src/components/MapView/MapView.tsx (hunk 3 above)
Comment (thaddmt) [reply packages/react-geo/src/components/MapView/MapView.tsx:10]: Yep, previously the shape of the Amplify configuration object was not exported by Amplify, but now they are!

**Review by calebpollman — APPROVED** [review]
🚢

**Pull request merged**